Xamarin.Forms.Platform.Android/Material/MaterialActivityIndicatorRenderer.cs

```diff
@@ -40,9 +40,16 @@ public MaterialActivityIndicatorRenderer(Context context)
 		{
 			VisualElement.VerifyVisualFlagEnabled();
 
-			_control = new AProgressBar(new ContextThemeWrapper(context, Resource.Style.XamarinFormsMaterialProgressBarCircular), null, Resource.Style.XamarinFormsMaterialProgressBarCircular);
-			_control.Indeterminate = true;
-			AddView(_control);
+			_control = new AProgressBar(new ContextThemeWrapper(context, Resource.Style.XamarinFormsMaterialProgressBarCircular), null, Resource.Style.XamarinFormsMaterialProgressBarCircular)
+			{
+				Indeterminate = true
+			};
+			var squareLayout = new SquareLayout(context)
+			{
+				LayoutParameters = new LayoutParams(LayoutParams.MatchParent, LayoutParams.MatchParent, GravityFlags.Center)
```

Comment:
Maybe instead of making a new layout, make this renderer inherit from the square layout. I am not sure if that will work, but then we can further reduce nesting.
Or, instead of using AProgress directly here, make a SquareProgress control.

Reply:
done.